tensorflow/stream_executor/rocm/rocm_driver.cc

```diff
@@ -490,9 +490,8 @@ ROCMDriver::DeviceGetSharedMemConfig(int device_ordinal) {
   // FIXME - need to set a 32-bit value here
   unsigned char valueC = static_cast<unsigned char>(value);
   uint32_t value32 = (valueC << 24) | (valueC << 16) | (valueC << 8) | (valueC) ;
-  assert (value32 == value); // if mismatch this indicates case where hipMemsetAsyc can't emulate hipMemSetD32
```

Comment:
@sumesh13 not so sure the rationale behind changes in this file, could you help clarify?

Reply:
let me dig up the example to see why I did that ..

Reply:
Let me bring this change in a different PR while I look for the test case